### ISSUE TITLE
Robots don't render items properly +  Other misc Robot issues

### ISSUE THREAD

**Issue by LuigiHutch** [AlexIIL/Buildcraft_1.8_Issues#91]
Forge 11.15.0.1688
BC 7.2.0 Pre 6

Robots holding items weirdly:
![2016-01-11_21 45 27](https://cloud.githubusercontent.com/assets/9326429/12248057/d4c830ac-b8ae-11e5-8965-d3f62fd636f0.png)

Robots not doing anything with a normal gate set up. Also the map location has render issues when placed inside a gate gui.
![2016-01-11_22 04 59](https://cloud.githubusercontent.com/assets/9326429/12248137/60279174-b8af-11e5-9855-01895f0d031c.png)

**Comment by LuigiHutch** [comment]
So actually robots do work properly apart from the planter which works once in a million years :/

I thought they weren't working but I was missing the go to station action

**Comment by AlexIIL** [comment]
So this is just 2 issues:
- [x] Robots not rendering items properly
- [ ] Robot planter too deep in thought to work for human

**Comment by LuigiHutch** [comment]
Correct :)

**Comment by Okimo** [comment]
So for me planter works fine but Delivering robot on requests don't want to work.
Same problem with rendering map and rendering inventory of planter.
Also looks like Zone Planner is off center and i have to find my location.

**Comment by LuigiHutch** [comment]
Well I did a test on all the robots and here are a list of issues I found:
- Planters do not plant
- Tank and pump robots don't deposit fluids when an accept fluid action is available
- The Delivery Robot doesn't deliver when an request items action is available
- Miner robot doesn't mine lit redstone ore
- Bomber doesn't Bomb

Edit: these were all tested with a landmark bound working area

**Comment by Okimo** [comment]
So another thing for robots.
I tried to right click on robot with dia helmet and got rendering crash:
http://pastebin.com/ge8VVkzb

**Comment by Okimo** [comment]
And Another thing for robots.
When I quited game with robots on and then went back I got massive FPS drop down (well should call it SecondsPerFrame). When I commanded them to go back to station. 2 of them went totally missing another two didn't respond to settings of emerald gate next robot changed from planter to harvester and rest of them worked fine. When I had all of them stopped I had to relaunch the game to get rid of lag spikes.
